network/src/peers/peer/peer.rs

```diff
@@ -42,13 +49,21 @@ const FAILURE_THRESHOLD: usize = 5;
 
 impl Peer {
     pub fn new(address: SocketAddr, data: Option<&snarkos_storage::Peer>) -> Self {
+        let mut block_height = Default::default();
```

Comment:
```suggestion
        let mut block_height = 0;
```
this should be more self-explanatory for a plain number